src/logics/compound-v2/scripts/get-tokens-data.ts

```diff
@@ -12,8 +12,8 @@ const cTokenAddresses = [
   '0x7713DD9Ca933848F6819F38B8352D9A15EA73F67', // cFEI
   '0xFAce851a4921ce59e912d19329929CE6da6EB0c7', // cLINK
   '0x95b4eF2869eBD94BEb4eEE400a99824BF5DC325b', // cMKR
-  '0x158079Ee67Fce2f58472A96584A73C7Ab9AC95c1', // cREP
-  '0xF5DCe57282A584D2746FaF1593d3121Fcac444dC', // cSAI
+  // '0x158079Ee67Fce2f58472A96584A73C7Ab9AC95c1', // cREP Legacy
+  // '0xF5DCe57282A584D2746FaF1593d3121Fcac444dC', // cSAI Legacy
```

Comment:
Is it better to remove these lines rather than comment them out? We could add them back when we need them again.

Reply:
Sure, and I don't think we need to add them back, so I've removed those lines.